### ISSUE TITLE
Remove Goo.gl shortener

### ISSUE THREAD

**Issue by UVLabs** [Codeinwp/tweet-old-post#150]
According to this: https://developers.googleblog.com/2018/03/transitioning-google-url-shortener.html  we should remove goo.gl by May 30th and/or switch to Firebase Dynamic Links 

@ineagu @selul

**Comment by contactashish13** [comment]
PR: https://github.com/Codeinwp/tweet-old-post/pull/225

PS - I would suggest creating a test email account and then using that to create credentials for each shortener so that test cases can run instead of commenting them out.

**Comment by UVLabs** [comment]
@selul can you look into Ash's recommendation?

**Comment by selul** [comment]
@UVLabs yes, ash is right, we can grab some credentials for testing in order to have proper testing scenarios.
